linera-service/tests/end_to_end_tests.rs

```diff
@@ -161,10 +161,10 @@ impl AmmApp {
     }
 }
 
-#[cfg_attr(feature = "rocksdb", test_case(LocalNetTestingConfig::new(Database::RocksDb, Network::Grpc) ; "rocksdb_grpc"))]
-#[cfg_attr(feature = "scylladb", test_case(LocalNetTestingConfig::new(Database::ScyllaDb, Network::Grpc) ; "scylladb_grpc"))]
-#[cfg_attr(feature = "aws", test_case(LocalNetTestingConfig::new(Database::DynamoDb, Network::Grpc) ; "aws_grpc"))]
-#[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, None) ; "kubernetes_grpc"))]
```

Comment:
The issue with removing this is that for `macOS` users (or anyone really) that want to specify a specific binaries path for tests, they need to be able to pass that path here (was done by replacing None by the path).
If not then we have to cross compile from within the container, which takes forever and significantly slows down dev cycle if we're doing e2e test related stuff 😅

Reply:
Ah, I didn't know. I'll revert that, and add a comment.

Reply:
Done: https://github.com/linera-io/linera-protocol/pull/1622/commits/71008189bc6245c12a963cb5c8a229d0b1ecb758
I'm not sure I have enough context to add an intelligent comment here.
Shouldn't we at some point remove that argument anyway, and instead use another environment variable for it? (But please not in this PR! :sweat_smile:)

Reply:
We could! I guess per test control of this is not generally needed 🤔